lms/static/js/bookmarks/views/bookmarks_list_button.js

```diff
@@ -16,11 +16,17 @@
             },
 
             initialize: function () {
-                this.bookmarksListView = new BookmarksListView({
-                    collection: new BookmarksCollection(),
-                    loadingMessageView: new MessageView({el: $(this.loadingMessageElement)}),
-                    errorMessageView: new MessageView({el: $(this.errorMessageElement)})
-                });
+                var bookmarksCollection = new BookmarksCollection(
+                    {course_id: $('.courseware-results').data('courseId')}
```

Comment:
Again, I think it would be cleaner to pass this data in through factory js instead of HTML nodes.